Resources/doc/reference/action_create_edit.rst

```diff
@@ -14,13 +14,14 @@ of the fields and forms available in these views and any other relevant settings
 Basic configuration
 -------------------
 
-To do:
+.. note::
 
-- global (yml) options that affect the create and edit actions
-- a note about Routes and how disabling them disables the related action
-- using configureFormFields() to set which fields to display
-- options available when adding fields, inc custom templates
-- link to the field_types document for more details about specific field types
+    **TODO**:
```

Comment:
In case you don't know it, `rst` allows to define contents as "comments", so they are not output in the generated docs. Maybe this could be useful in this case of **TODO** lists.  To create a comment, you just need to write two dots at the beginning of the line: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#comments